rats-tls/samples/rats-tls-client/client.c

```diff
@@ -143,7 +143,12 @@ int rats_tls_client_startup(rats_tls_log_level_t log_level, char *attester_type,
 		goto err;
 	}
 
-	const char *msg = "Hello and welcome to RATS-TLS!\n";
+	const char *msg;
+	if (verdictd)
+		msg = "{ \"command\": \"echo\", \"data\": \"Hello and welcome to RATS-TLS!\" }";
```

Comment:
please add "n" behind the last "\" to keep align with the msg format with the original msg.